lang/en.inc.php

```diff
@@ -32,6 +34,7 @@
 $messages['label_authtimestamp'] = "Last authentication";
 $messages['label_businesscategory'] = "Business category";
 $messages['label_carlicense'] = "Car license";
+$messages['label_comment'] = "Comment";
```

Comment:
label_x is reserved to tranlsation of LDAP attributes, another key like "comment" and rename "comment" key to "insert_comment"